use-timescale/extensions/index.md

```diff
@@ -11,52 +11,137 @@ tags: [extensions]
+[bloom]: https://www.postgresql.org/docs/current/bloom.html
+[bool_plper]: https://www.postgresql.org/docs/current/plperl-funcs.html
+[btree_gin]: https://www.postgresql.org/docs/current/btree-gin.html
+[btree_gist]: https://www.postgresql.org/docs/current/btree-gist.html
+[citext]: https://www.postgresql.org/docs/current/citext.html
+[cube]: https://www.postgresql.org/docs/current/cube.html
+[dict_int]: https://www.postgresql.org/docs/current/dict-int.html
+[dict_xsyn]: https://www.postgresql.org/docs/current/dict-xsyn.html
+[earthdistance]: https://www.postgresql.org/docs/current/earthdistance.html
+[fuzzystrmatch]: https://www.postgresql.org/docs/current/fuzzystrmatch.html
+[hstore]: https://www.postgresql.org/docs/current/hstore.html
+[insert_username]: https://www.postgresql.org/docs/current/contrib-spi.html#CONTRIB-SPI-INSERT-USERNAME
+[intagg]: https://www.postgresql.org/docs/current/intagg.html
+[intarray]: https://www.postgresql.org/docs/current/intarray.html
+[isn]: https://www.postgresql.org/docs/current/isn.html
+[jsonb_plperl]: https://www.postgresql.org/docs/current/datatype-json.html#DATATYPE-JSON-TRANSFORMS
+[lo]: https://www.postgresql.org/docs/current/lo.html
+[ltree]: https://www.postgresql.org/docs/current/ltree.html
+[moddatetime]: https://www.postgresql.org/docs/current/contrib-spi.html#CONTRIB-SPI-MODDATETIME
+[old_snapshot]: https://www.postgresql.org/docs/16/oldsnapshot.html
+[pg_freespacemap]: https://www.postgresql.org/docs/current/pgfreespacemap.html
+[pg_prewarm]: https://www.postgresql.org/docs/current/pgprewarm.html
+[pg_stat_statements]: https://www.postgresql.org/docs/current/pgstatstatements.html
+[pg_trgm]: https://www.postgresql.org/docs/current/pgtrgm.html
+[pg_visibility]: https://www.postgresql.org/docs/current/pgvisibility.html
+[pgai]: /ai/:currentVersion:/
+[pgaudit]: https://www.pgaudit.org/
+[pgpcre]: https://github.com/petere/pgpcre
+[pgrouting]: https://pgrouting.org/
+[pgrowlocks]: https://www.postgresql.org/docs/current/pgrowlocks.html
+[pgstattuple]: https://www.postgresql.org/docs/current/pgstattuple.html
+[pgvector]: https://github.com/pgvector/pgvector
+[pgvectorscale]: https://github.com/timescale/pgvectorscale
+[plperl]: https://www.postgresql.org/docs/current/plperl.html
+[plpgsql]: https://www.postgresql.org/docs/current/plpgsql.html
 [postgis]: /use-timescale/:currentVersion:/extensions/postgis/
-[postgres-fdw]: /use-timescale/:currentVersion:/schema-management/foreign-data-wrappers/
\ No newline at end of file
+[postgis_raster]: https://postgis.net/docs/RT_reference.html
+[postgis_sfcgal]: https://postgis.net/docs/reference_sfcgal.html
+[postgis_tiger_geocoder]: https://postgis.net/docs/Extras.html#Tiger_Geocoder
+[postgis_topology]: https://postgis.net/workshops/postgis-intro/topology.html
+[refint]: https://postgrespro.com/docs/postgresql/9.6/contrib-spi#idp144721
+[seg]: https://www.postgresql.org/docs/current/seg.html
+[pgcrypto]: /use-timescale/:currentVersion:/extensions/pgcrypto/
```

Comment:
this link seems wrong, it's also a built-in:
https://www.postgresql.org/docs/current/pgcrypto.html